dist/pythonlibs/testrunner/__init__.py

```diff
@@ -15,8 +15,14 @@
 from .spawn import find_exc_origin, setup_child, teardown_child
 from .unittest import PexpectTestCase   # noqa, F401 expose to users
 
+# Timeout for tests can be changed by setting RIOT_TEST_TIMEOUT in the
+# environment variables which may be required for some systems that take
```

Comment:
please adapt this comment to be more generic, i.e. not stating that this a used to work around flash or reset problems.

Reply:
@smlng Made the change